### PR TITLE
 Changed the color for the underlining to be gray instead of purple

### PR DESCRIPTION
**Pull request by climberfields**

### CODE REVIEW

File: designsafe/static/scripts/data-depot/components/projects/publication-preview/publication-preview.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
     .preview-tree-btn-wrapper {
         position: absolute;
         left: -175px;
-        border-bottom: 3px solid purple;
+        border-bottom: 3px solid gray;
```
Comment (Strmiska) [inline designsafe/static/scripts/data-depot/components/projects/publication-preview/publication-preview.scss]: I think this was included in one of your other PRs

**Comment by rstijerina** [comment]
Is there a task for this?

File: designsafe/static/scripts/data-depot/components/projects/publication-preview/publication-preview.scss (hunk 1 above)
Comment (climberfields) [reply designsafe/static/scripts/data-depot/components/projects/publication-preview/publication-preview.scss]: This is supposed to be changed to gray. Is this doing something else?

**Comment by rstijerina** [comment]
Closing - this was incorporated into https://github.com/DesignSafe-CI/portal/pull/493

**Pull request closed without merging**